src/app/core/models/templates/tester-response-template.model.ts

```diff
@@ -14,6 +14,8 @@ export class TesterResponseTemplate extends Template {
   teamResponse: Section;
   testerResponse: TesterResponseSection;
   comment: IssueComment;
+  teamChosenSeverity: string;
```

Comment:
Just a minor nit, but since the `teamChosenSeverity` and `teamChosenType` attributes in the `TeamResponseSection` class are optional and could be undefined, or have an empty string returned from the `extractStringBetween` method, would it be better if those two attributes in this `TesterResponseTemplate` have the optional attribute type as well and only assign them values if `testerResponse.getTeamChosenSeverity()` (or getChosenType) returns a non empty string?

Reply:
Hey sure, i think it will be better if we do that